### ISSUE TITLE
github-token vs ghToken

### ISSUE THREAD

**Issue by forresto** [outsideris/potential-conflicts-checker-action#4]
Inputs want `github-token`:

https://github.com/outsideris/potential-conflicts-checker-action/blob/5b014d8352271d30e37df032eb03f0b9123b9659/action.yml#L3-L5

But Readme and code expect `ghToken`:

https://github.com/outsideris/potential-conflicts-checker-action/blob/86c9d2ffbfad8c21821c3b25928b2a9bab9cd9f2/README.md#L25

https://github.com/outsideris/potential-conflicts-checker-action/blob/9b77a916876b66d3b7596ddd9e1e6ef170bdd32d/index.js#L21

When following Readme, warning looks like this:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/395307/91425421-1ca37900-e829-11ea-948b-61670aedc763.png">

**Comment by stovmascript** [comment]
Yeah, looks like `action.yml` input should be renamed `github-token` -> `ghToken`.
